nextflow.config

```diff
@@ -11,6 +11,7 @@ params {
 
     // Input options
     input                      = null
+    platform                   = null
```

Comment:
so this makes it so that the user always has to specify this in their command line etc or else the pipeline will fail? I like that design decision, just making sure i'm interpreting it correctly

Reply:
yes